.circleci/config.yml

```diff
@@ -356,16 +356,6 @@ jobs:
           command: |
             RUST_BACKTRACE=1 $CI_TIMEOUT \
             ./scripts/run_quarantined.sh -c <your package here> -r 3 -f
-  validate-cluster-test-dockerfile:
```

Comment:
Thanks for reworking the Cluster Test Dockerfile workflow and retiring this check in CI.